nodejs/week3/warmup/index.js

```diff
@@ -0,0 +1,103 @@
+const express = require('express');
+const app = express();
+
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
+
+app.get('/calculator/add', getAddition);
+app.get('/calculator/multiply', getMultiplication);
+app.get('/calculator/subtraction', getSubtraction);
+app.get('/calculator/division', getDivision);
```

Comment:
Nice! You're missing thought the [POST](https://github.com/HackYourFuture-CPH/node.js/tree/main/week3/homework-template#getting-data-through-the-request-body-using-post) requests!
 What you have implemented is the get requests, meaning you will get the parameters from the url and the req.query. You also have to implement two post requests and in that case you will get the parameters from the request's body and the req.body instead of req.query